theme/utils.py

```diff
@@ -82,3 +83,19 @@ def get_quota_message(user):
                                                               zone=uq.zone,
                                                               percent=rounded_percent)
         return return_msg
+
+
+def get_user_from_username_or_email(username_or_email, **kwargs):
```

Comment:
@sblack-usu all I did was type "def get_user_from_username_or_email" and CoPilot wrote this function... :)